diode-server/reconciler/logs_retriever.go

```diff
@@ -81,10 +82,59 @@ func decodeBase64ToInt64(encoded string) (int64, error) {
 	return num, nil
 }
 
+func retrieveIngestionStatsSummary(ctx context.Context, client RedisClient) (*reconcilerpb.RetrieveIngestionLogsResponse, error) {
+
+	pipe := client.Pipeline()
+
+	results := make([]*redis.Cmd, 0)
+	results = append(results, pipe.Do(ctx, "FT.SEARCH", "ingest-entity", "*", "LIMIT", 0, 0))
+	for i := 1; i < 5; i++ {
```

Comment:
Some comment would be useful here, what are we doing here and why 5? Assuming ingestion state `NEW`, we can also use existing code generate from proto `reconcilerpb.State_NO_CHANGES` (4)

Reply:
agree

Reply:
agree to change it